### PR TITLE
parser: Check for comma after parameter in function call

### PR DESCRIPTION
**Pull request by f1ypopper**

### CODE REVIEW

File: src/parser.rs [hunk 1]
```diff
@@ -4518,8 +4518,15 @@ pub fn parse_call(tokens: &[Token], index: &mut usize) -> (ParsedCall, Option<Ja
                     TokenContents::Comma => {
                         // Treat comma as whitespace? Might require them in the future
                         *index += 1;
+                        next_parameter = true;
                     }
                     _ => {
+                        if !next_parameter {
+                            error = error.or(Some(JaktError::ParserError(
+                                "Invalid syntax".to_string(),
```
Comment (jntrnr) [inline src/parser.rs:4526]: Can we give a better error here? What makes the syntax invalid?

**Comment by f1ypopper** [comment]
Hey @jntrnr can you please review the PR again?

**Comment by jntrnr** [comment]
@f1ypopper - looks like you'll need to follow the submission guidelines that BuggieBot links to.

Also, you should add a test to show that the error is working

**Comment by AtkinsSJ** [comment]
Closed as this is a change for the now-gone Rust-based compiler. Feel free to port your changes to the new compiler, if they're not already added, and open a new PR. :^)

**Pull request closed without merging**